tool/site/client/src/services/test-packs/index.ts

```diff
@@ -52,38 +63,47 @@ export const createPrescriptionsFromExcelFile = (
 }
 
 function createPrescriptions(
-  prescriptionType: PrescriptionType,
-  patients: Array<fhir.BundleEntry>,
-  prescribers: Array<fhir.BundleEntry>,
-  places: Array<fhir.BundleEntry>,
-  rows: Array<PrescriptionRow>,
+  patientRows: Map<string, PatientRow>,
```

Comment:
very nice improvements here :+1:

Reply:
thanks, I've separated the concerns for data population and building fhir resources as they were previously mixed in various functions before which made it very hard to follow